src/adagrams.js

```diff
@@ -1,15 +1,97 @@
+// WAVE 1
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterPool = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12,'F': 2,
+    'G': 3, 'H': 2, 'I': 9, 'J': 1, 'K': 1, 'L': 4, 
+    'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6,
+    'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1,
+    'Y': 2, 'Z': 1
+  };
+  
+  let letters = [];
+  for (let letter in letterPool) {
+    for (let i = 0; i < letterPool[letter]; i++) {
+      letters.push(letter);
+    }
+  }
+
+  let hand = [];
+  for (let i = 0; i < 10; i++) {
+    let randomIndex = Math.floor(Math.random() * letters.length);
```

Comment:
Great work getting comfortable with JavaScript's syntax. It looks like you're getting use to looping over data and adding conditional logic ✅